test_conformance/conversions/conversions_data_info.h

```diff
@@ -159,9 +177,9 @@ DataInfoSpec<InType, OutType>::DataInfoSpec(const DataInitInfo &agg)
 
         InType eps = std::is_same<InType, cl_float>::value ? (InType) FLT_EPSILON : (InType) DBL_EPSILON;
         if (std::is_integral<OutType>::value)
-        { // to char/uchar/short/ushort/int/uint/long/ulong
+        { // to char/uchar/short/ushort/int/uint/long/ulong/half
             if (sizeof(OutType)<=sizeof(cl_short))
-            { // to char/uchar/short/ushort
+            { // to char/uchar/short/ushort/half
```

Comment:
I'm a little skeptical this will work for half as an InType or as an OutType:

* For half an InType, note that `std::is_floating_point<InType>::value` will be false, so this will need to be handled specially.
* For half as an OutType, note that `std::is_integral<OutType>::value` will be true, so this will need to be handled specially, also.

Reply:
Done, separate checks for both `InType` and `OutType` added to handle `cl_half` impact on `clamp_ranges`